apps/meteor/app/livechat/imports/server/rest/rooms.js

```diff
@@ -19,9 +19,9 @@ const validateDateParams = (property, date) => {
 
 API.v1.addRoute(
 	'livechat/rooms',
-	{ authRequired: true, permissionsRequired: { GET: { permissions: ['view-livechat-rooms', 'view-l-room'], operation: 'hasAny' } } },
```

Comment:
Note: Removing this because there's already a permission check within this endpoint in place - so avoiding multiple checks